Sensei Onboarding - Fix the Sensei Bundle price.

dadish opened the pull request:
#### Proposed Changes

* Fixes the Sensei Bundle pricing on the Sensei Plans screen.

#### Testing Instructions

<!--
Add as many details as possible to help others reproduce the issue and test the fix.
"Before / After" screenshots can also be very helpful when the change is visual.
-->

* Start the Sensei Flow.
* Get to the Sensei Plans screen.
* Confirm that Sensei Bundle costs $54 for Monthly plan.
* Confirm that Sensei Bundle costs $449 for Yearly plan.

#### Before

<img width="1728" alt="Screen Shot 2022-12-05 at 18 31 02" src="https://user-images.githubusercontent.com/2578542/205664411-340cdd16-7e74-4492-9805-19a9bc5f11b6.png">
<img width="1728" alt="Screen Shot 2022-12-05 at 18 31 15" src="https://user-images.githubusercontent.com/2578542/205664433-bf23b4de-6ff2-421b-808a-30bee6a9aac1.png">


#### After

<img width="1728" alt="Screen Shot 2022-12-05 at 18 27 51" src="https://user-images.githubusercontent.com/2578542/205664499-30832521-e7a0-4ba5-8725-2264fc3d7370.png">
<img width="1727" alt="Screen Shot 2022-12-05 at 18 28 01" src="https://user-images.githubusercontent.com/2578542/205664519-6137ec58-666a-4dbe-aa88-dee203737fde.png">


#### Pre-merge Checklist

<!--
Complete applicable items on this checklist **before** merging into trunk. Inapplicable items can be left unchecked.

Both the PR author and reviewer are responsible for ensuring the checklist is completed.
-->

- [ ] [Have you written new tests](https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md) for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?
<!--
Link a related issue to this PR. If the PR does not immediately resolve the issue,
for example, it requires a separate deployment to production, avoid
using the "fixes" keyword and instead attach the [Status] Fix Inbound label to
the linked issue.
-->

Related to https://github.com/Automattic/sensei-pro/issues/1952

Commit:
Sensei Onboarding - Fix the Sensei Bundle price.

mikeyarce commented:
I can confirm the price is correct in the plan screen and at checkout! 🎉 

A few questions:
- It's no longer giving me the Canadian dollars `C` even though it appears it is in Canadian dollars.
- I noticed the "Sensei Pro Bundle" text is missing, is that on purpose?
- I also noticed the new price text is a gray colour - I think the contrast in black was better personally
<img width="975" alt="Screenshot 2022-12-06 at 2 59 18 PM" src="https://user-images.githubusercontent.com/3220162/206042902-1f534f5f-0cc3-4bc4-912a-24a40d40b64f.png">

Commit:
Sensei Onboarding - Fix Sensei Plan pricing currency and styles.

dadish commented:
Thanks for the review @mikeyarce !

> I can confirm the price is correct in the plan screen and at checkout! 🎉
🎉 

>     * It's no longer giving me the Canadian dollars `C` even though it appears it is in Canadian dollars.
Good catch! Fixed. I wonder if we would've caught that if you didn't have your currency set to Canadian. 😄 We got lucky!

>     * I noticed the "Sensei Pro Bundle" text is missing, is that on purpose?
Not on purpose. I somehow missed that part. Added it back. 💪 

>     * I also noticed the new price text is a gray colour - I think the contrast in black was better personally
Agree. Updated.


744cc0fb4a89ef86337d65599d44008e9420f261

mikeyarce reviewed (approved):
Looks great, thanks @dadish !  🎉

a8ci18n commented:
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7683371

Thank you @dadish for including a screenshot in the description! This is really helpful for our translators.

a8ci18n commented:
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7683371).
